### DISCUSSION TITLE
Can't Update Metadata for Solana Token

### DISCUSSION THREAD

**Discussion by ccstroescu** [vercel/community#4073] [Help]
### Page to Investigate

https://token-creator-lac.vercel.app/update

### Steps to Reproduce

Hello,
When I go to https://token-creator-lac.vercel.app/update for updating Solana token metadata it doesn't work. Getting lots of errors in browser console:
Uncaught (in promise) Error: User rejected the request.
    at Es.<anonymous> (<anonymous>:3:208709)
    at Generator.next (<anonymous>)
    at i (<anonymous>:1:1499) 

and a Transaction Expired window when I click Update Metadata button.
This happens for both me an my colleague so is not a local issue.

Also on upload metadata we get error: Uncaught (in promise) Error: failed to get recent blockhash: Error: 403 :  {"jsonrpc":"2.0","error":{"code": 403, "message":"Access forbidden, contact your app developer or support@rpcpool.com."}, "id": "8d8ef1cc-a873-4c94-88b3-87b55e3adff6" } 

We need to know how to update the Solana token metadata since vercel.app does not work.

**Comment by amyegan** [comment]
Hi @sclitifyitcom. Are you able to share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so we can help with debugging?

**Comment by ccstroescu** [comment]


**Reply by FigrHed** [reply]
having the same problem

**Reply by doctorbinsibar** [reply]
yes

**Comment by airose007** [comment]
If you looking for help with building a sniper bot, join here https://t.me/+U_BCU0XDRYU0ZWQy
